src/Facade/Services/IPurchaseOrderReportFacadeService.cs

```diff
@@ -11,31 +11,24 @@
     public interface IPurchaseOrderReportFacadeService
     {
         IResult<ReportReturnResource> GetOrdersBySupplierReport(
-            OrdersBySupplierSearchResource resource,
-            IEnumerable<string> privileges);
+            OrdersBySupplierSearchResource resource);
 
         IEnumerable<IEnumerable<string>> GetOrdersBySupplierExport(
-            OrdersBySupplierSearchResource resource,
-            IEnumerable<string> privileges);
```

Comment:
was there any need for all of these methods to tkae a privileges list?

Reply:
probably nothing in purchasing really needs it so far. There will definitely be finance reports etc that are restricted. Don't know if they will ever need the privileges passed through to the facade though